devops/templates/create-env-template.yml

```diff
@@ -1,26 +1,50 @@
 # install needed packages
+parameters:
+- name: pythonVersion
+  type: string
+- name: condaEnv
+  type: string
+  default: interpret_conda_env
+
 steps:
-  - bash: conda create --yes --quiet --name interp python=3.6
+  - bash: brew install libomp
+    displayName: Use Homebrew to install libomp on MacOS
+    condition:  eq(variables['Agent.OS'], 'Darwin')
+
+  - bash: conda create --yes --quiet --name ${{parameters.condaEnv}} python=${{parameters.pythonVersion}}
     displayName: Create Anaconda environment
 
   - bash: |
-      source activate interp
-      conda install --yes --quiet --name interp pytorch-cpu torchvision-cpu -c pytorch
-      conda install --yes -c anaconda -n interp jupyter pip
+      source activate ${{parameters.condaEnv}}
+      conda install --yes --quiet --name  ${{parameters.condaEnv}} pytorch-cpu torchvision-cpu -c pytorch
+      conda install --yes -c anaconda -n  ${{parameters.condaEnv}} jupyter pip
     displayName: Install Anaconda packages
 
   - bash: |
-      source activate interp
+      source activate  ${{parameters.condaEnv}}
+      conda install --yes -c conda-forge lightgbm
+    displayName: Install lightgbm from conda on MacOS
+    condition:  eq(variables['Agent.OS'], 'Darwin')
+
+  - bash: |
+      source activate ${{parameters.condaEnv}}
+      pip install torch
+      pip install torchvision
+    displayName: Install torch from pip on MacOS
+    condition:  eq(variables['Agent.OS'], 'Darwin')
```

Comment:
These MacOS-only installations make me nervous.

Reply:
Unfortunately some of them are necessary because the lower-level components of libraries like pytorch need different installs.

Reply:
Hmm, yeah, those packages definitely still need to be installed for other operating systems

Reply:
If you look at some of the other tasks, they are.